mineinabyss-components/src/main/kotlin/com/mineinabyss/components/mobs/Bird.kt

```diff
@@ -0,0 +1,8 @@
+package com.mineinabyss.components.mobs
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("mineinabyss:bird")
+class Bird
```

Comment:
This should be done as a category component or something similar in geary-addons